homeassistant/components/zwave_js/__init__.py

```diff
@@ -309,30 +310,34 @@ def async_on_value_updated(
             return
 
         device = dev_reg.async_get_device({get_device_id(client, value.node)})
-        event_data = {
-            ATTR_DOMAIN: DOMAIN,
-            ATTR_NODE_ID: value.node.node_id,
-            ATTR_HOME_ID: client.driver.controller.home_id,
-            ATTR_DEVICE_ID: device.id,  # type: ignore
-            ATTR_COMMAND_CLASS: value.command_class,
-            ATTR_COMMAND_CLASS_NAME: value.command_class_name,
-            ATTR_ENDPOINT: value.endpoint,
-            ATTR_PROPERTY: value.property_,
-            ATTR_PROPERTY_NAME: value.property_name,
-            ATTR_PROPERTY_KEY: value.property_key,
-            ATTR_PROPERTY_KEY_NAME: value.property_key_name,
-            ATTR_VALUE: value.value,
-        }
 
         unique_id = get_unique_id(
             client.driver.controller.home_id, disc_info.primary_value.value_id
         )
-        if entity_id := ent_reg.async_get_entity_id(
-            disc_info.platform, DOMAIN, unique_id
-        ):
-            event_data[ATTR_ENTITY_ID] = entity_id
+        entity_id = ent_reg.async_get_entity_id(disc_info.platform, DOMAIN, unique_id)
+
+        raw_value = value = value.value
```

Comment:
Looks like we're overwriting the `value` variable with another type here.

Reply:
good catch